cmd/trtl/main.go

```diff
@@ -268,8 +321,37 @@ func main() {
 
 // serve starts the trtl server and blocks until it is stopped.
 func serve(c *cli.Context) (err error) {
-	// TODO: invoke replica server
+	var conf config.Config
+	var db *honu.DB
+
+	replicaConfig := honuconfig.ReplicaConfig{
+		PID:    c.Uint64("pid"),
+		Region: c.String("region"),
+	}
+	if db, err = honu.Open(c.String("db"), replicaConfig); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	var server *trtl.Server
+	conf = config.Config{
+		Enabled:  true,
+		BindAddr: c.String("bindaddr"),
+	}
+	if server, err = trtl.New(db, conf); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	if err = server.Serve(); err != nil {
+		return cli.Exit(err, 1)
+	}
+
 	fmt.Println("started the trtl data replication service")
+
+	// Block until the server is stopped.
+	if err = server.BlockUntilShutdown(); err != nil {
+		return cli.Exit(err, 1)
+	}
+	fmt.Println("server terminated")
```

Comment:
The `Serve` method should block until shutdown; I'll pull your PR and make these updates.